### ISSUE TITLE
[Error handling] Removing a non-existent key doesn't return any error

### ISSUE THREAD

**Issue by DivyatejaChe** [emeraldsanto/react-native-encrypted-storage#52]
I am trying to handle success & failure scenarios when an already stored key is being removed.

EXPECTED: _removeSensitiveInfo()_ would return error if the key being removed doesn't exist in Keychain
CURRENT: Doesn't return any error and try block executes successfully

> This is my _removeSensitiveInfo()_ implementation as suggested by README
```
export const removeSensitiveInfo = async key => {
  try {
    const result = await EncryptedStorage.removeItem(key)
    return result
  } catch (error) {
    console.error('Encountered error while removing data: ', error)
  }
}
```

This is my component calling _removeSensitiveInfo()_ when user tries to logout.
```
await removeSensitiveInfo('user_session').then(() => {
        navigation.reset({
          index: 0,
          routes: [{ name: 'Home' }]
        })
        dispatch({ type: 'DESTROY_STATE' })
      })
```

**Comment by flora8984461** [comment]
This seems to happen on android. I've tried both, using ios on Mac, and using android on windows, Android bypass the error, but IOS catches it.

**Comment by micky2be** [comment]
I tend to disagree about throwing an error. I know several APIs are implementing it this way but IMHO, if what you try to delete doesn't exist in the first place, the result is successful. Aka the data is not there.
